src/Datasource/EntityTrait.php

```diff
@@ -394,7 +394,7 @@ public function isEmpty(string $field): bool
                 empty($value) ||
                 (
                     is_string($value) &&
-                    empty($value)
+                    strlen($value) === 0
```

Comment:
```suggestion
                    $value === ''
```

Reply:
@mtak3 You need to make this change before we can merge. I would make it, but you didn't allow maintainers to make changes.

Reply:
I'll fix this up post merge.